app/controllers/rounds_controller.rb

```diff
@@ -36,6 +36,7 @@ def pairings_data
             pairings: round.pairings.map { |pairing| {
               id: pairing.id,
               table_number: pairing.table_number,
+              is_single_sided_swiss: @tournament.swiss_format == :single_sided.to_s,
```

Comment:
It might be less clutter and more correct if we put this on the stage or the round, and use the stage format instead of the tournament format?

Reply:
I moved it to the stage.  Good call.